doc/howto/build-linux.md

```diff
@@ -0,0 +1,105 @@
+## Build and configure X-Ray 1.6 on Linux
+
+Make sure you are using STALKER Call Of Pripyat v1.6.02 and your system contains the following packages:
+git, cmake, GLEW, Freeimage, Lockfile, Openal, TBB, Crypto++, pugixml, Theora, Ogg, SDL2, Lzo, Jpeg.
+
+To install software above in **Debian** and **Ubuntu** derivatives just run the command:
+
+```
+sudo apt install git cmake libglew-dev libfreeimage-dev liblockfile-dev libopenal-dev libtbb-dev libcrypto ++ - dev libpugixml-dev libogg-dev libtheora-dev libvorbis-dev libsdl2-dev liblzo-dev-libjdg2-dev
+```
+
+In case you are **Fedora** user:
+```
+sudo dnf install git cmake glew-devel freeimage-devel liblockfile-devel openal-devel tbb-devel cryptopp-devel pugixml-devel libogg-devel libtheora-devel libvorbis-devel SDL2-devel lzo-devel libjpeg-turbo-devel
+```
+Next clone the repo with its submodules:
+```
+git clone https://github.com/OpenXRay/xray-16.git && git submodule update --init --recursive
+```
+After the cloning is complete, the xray-16 directory should appear in the directory you invoked command above.
+Next it is needed to create directory for binary output and navigate there:
+```
+cd xray-16 && mkdir bin && cd bin
+```
+Please pay attention to the errors in next stage. Since they can occur in case of absence of any components.
+So, prepare your source to be compiled with:
+```
+cmake ..
+```
+You can customize build passing arguments to cmake.
+To enable debugging:
+```
+cmake .. -DCMAKE_BUILD_TYPE = RelWithDebInfo # The string with the debug flag is no longer supported
+```
+To use clang:
+```
+CC=clang CXX=clang++ cmake ..
+```
+To include all optimizations for your machine:
+```
+CFLAGS="-march=native" CXXFLAGS="-march=native" cmake ..
+```
+Finally
+```
+make -jX
+```
+Where X your processor's number of cores
+Also, to output the log and errors to different files:
+```
+make -jX > out.log 2 > error.log
+```
+To use linux binaries with the game we need to create directory `bin-linux` there.
+So, once compilation is done in the `xray-16/bin` directory execute the command:
+```
+make DESTDIR=/path/to/your/SCOP/bin-linux install
+```
+And the `bin-linux` directory will appear where the game is located.
+Yes, `/path/to/your/SCOP` it's the path to your legal copy of STALKER CoP.
+
+Backup `res` folder and `fsgame.ltx` in the original game folder in the form you obtained by installing the game from Steam, GOG or license disc
+Next copy from `xray-16` `res` folder and `fsgame.ltx` file to the `/path/to/your/SCOP`.
+
+To run the game invoke one of the following commands:
+If you want to debug the game, you need to prevent input capture:
+```
+setxkbmap -option grab: break_actions
```

Comment:
`setxkbmap -option grab:break_actions` - no space between `grab:` and `break_action`